components/homepage/MySkills.tsx

```diff
@@ -11,18 +11,22 @@ const MySkills = (props: Props) => {
           My Skills
         </h2>
         <div className="tech md:techlg flex md:flex-row">
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
-          <SkillTile imageUrl="" />
+          <SkillTile imageUrl="/js.svg" />
+          <SkillTile imageUrl="/react.svg" />
+          <SkillTile imageUrl="/Redux.svg" />
+          <SkillTile imageUrl="Next.svg" />
+          <SkillTile imageUrl="ts.svg" />
+          <SkillTile imageUrl="html.svg" />
+          <SkillTile imageUrl="css.svg" />
+          <SkillTile imageUrl="sass.svg" />
+          <SkillTile imageUrl="Material.svg" />
+          <SkillTile imageUrl="tailwind.svg" />
+          <SkillTile imageUrl="Motion.svg" />
+          <SkillTile imageUrl="git.svg" />
+          <SkillTile imageUrl="github.svg" />
+          <SkillTile imageUrl="nodejs.svg" />
+          <SkillTile imageUrl="ex.svg" />
+          <SkillTile imageUrl="mongodb.svg" />
```

Comment:
Why can't this go into a json file that you can loop over? That way when sanity is integrated you can pull from sanity instead of the json file and change the content of the page without editing the code.